lib/mayaUsd/ufe/UsdObject3d.h

```diff
@@ -42,6 +42,10 @@ class MAYAUSD_CORE_PUBLIC UsdObject3d : public Ufe::Object3d
     bool                visibility() const override;
     void                setVisibility(bool vis) override;
 
+#if UFE_PREVIEW_VERSION_NUM >= 2034
+    Ufe::UndoableCommand::Ptr setVisibleCmd(bool vis) override;
+#endif
+
```

Comment:
Added guard around "setVisibleCmd" since this feature is available in UFE V.2034 or later.